core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -69,13 +69,13 @@ private TableProperties() {
   public static final long COMMIT_STATUS_CHECKS_MIN_WAIT_MS_DEFAULT = 1000L; // 1s
 
   public static final String COMMIT_STATUS_CHECKS_MAX_WAIT_MS = "commit.status-check.max-wait-ms";
-  public static final long COMMIT_STATUS_CHECKS_MAX_WAIT_MS_DEFAULT = 60000L; // 1 minute
+  public static final long COMMIT_STATUS_CHECKS_MAX_WAIT_MS_DEFAULT = 60 * 1000L; // 1 minute
```

Comment:
Not sure why we included the L here, this is the smallest multiplication we do among these literals :) I think we could just remove it now, or maybe we should add L on all the constants if we are afraid of overflows ... Doesn't really matter to me

Reply:
Agree, let me drop L here and in one more place like this for consistency.

Reply:
Done.